clarity/contracts/arkadiko-swap-v1-1.clar

```diff
@@ -9,6 +9,8 @@
 (define-constant ERR-INVALID-LIQUIDITY u202)
```

Comment:
Should we update the name to v2, and make sure that all the other contracts are using the right version?
Looking at the code, the method `payoff-vault-debt` in `arkadiko-stacker-payer-v1-1` is has a hard coded reference to `arkadiko-swap-v1-1`, there could be a cascade there.

Reply:
Yes - in fact we are copying all this code into a `swap-v2.clar` file once it's ready for deployment, and then we'll be retiring the `arkadiko-stacker-payer-v1-1` in favor of a contract where the users claim the yield themselves instead of us putting the yield straight into the vault (since it was not scalable). SEe here https://github.com/arkadiko-dao/arkadiko/pull/319